### DISCUSSION TITLE
建议：能否弄一个不依赖于android的，纯java或kotlin的库, 想要在kotlin 桌面端使用

### DISCUSSION THREAD

**Discussion by sunny-zhang-98** [Tencent/MMKV#1026] [Ideas]
想要在kotlin 桌面端使用

**Comment by lingol** [comment]
I did not aware that there are people using Kotlin to write PC Apps. 
Anyway, it's possible to achieve that, in theory. But I highly doubt there will be fewer users developing PC Apps in Kotlin or Java. For the time being, we will stick to the C++ solution on PC.

**Comment by sunny-zhang-98** [comment]
@lingol You can look about Jetpack compose for desktop

**Comment by lingol** [comment]
The C++ solution is robust. Well, you are welcome to wrap one based on that for your favorite SDK/platform such as Jetpack compost for desktop or Javalet or flutter, or others.

**Comment by KunMinX** [comment]
事实上 mmkv 对 android 的依赖主要体现于对 Context 和 ContentProvider 的使用，context 主要是获取缓存目录和 contentResolver，使用率也不大，自行将这些成分剥离即可。

https://github.com/ctripcorp/mmkv-kotlin
